libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.scss

```diff
@@ -0,0 +1,63 @@
+/* Default styles for larger screens */
+.image-text-banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background: #dbc2f5 0% 0% no-repeat padding-box;
```

Comment:
For reuse of colors, use CSS variables specified in elements/theming/_variables.scss when specifying colors/background colors

Reply:
@AnnaL001 *resolved